tasks/translation-ollama/model.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+import logging
+import subprocess
+import time
+
+from ollama import ResponseError
+
+import requests
+import ollama
+
+
+OLLAMA_SERVER_URL = "http://127.0.0.1:11434"
+LLM = "llama3"
+
+
+def isOllamaInstalled() -> bool:
+    try:
+        subprocess.run(["ollama", "--version"], check = True, stdout = subprocess.PIPE, stderr = subprocess.PIPE)
+        return True
+    except (subprocess.CalledProcessError, FileNotFoundError):
+        return False
+
+
+def installOllama() -> None:
+    try:
+        subprocess.run("curl https://ollama.ai/install.sh | sh", shell = True, check = True)
+        logging.info(">> [OllamaRAG] Ollama installation was successful")
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f">> [OllamaRAG] An error occurred during Ollama installation: {e}")
+
+
+def isOllamaServerActiveAndRunning() -> bool:
+    try:
+        response = requests.get(OLLAMA_SERVER_URL)
+        return bool(response.ok)
```

Comment:
There is no need to cast it into `bool()`, `response.ok` is already a boolean value